src/test/java/com/softserveinc/ita/andrewobitotski/GoogleSearchTest.java

```diff
@@ -0,0 +1,26 @@
+package com.softserveinc.ita.andrewobitotski;
+
+import com.softserveinc.ita.pageobjects.andrewobitotski.GoogleSearchPage;
+import io.qameta.allure.Description;
+import org.testng.annotations.Test;
+
+import static com.softserveinc.ita.utils.ReadDataFileValues.MENTION;
+import static com.softserveinc.ita.utils.ReadDataFileValues.SEARCH_REQUEST;
+import static org.testng.Assert.assertTrue;
+
+public class GoogleSearchTest extends TestRunner {
+
+    @Test
+    @Description("Verify that search produces search result which contain expected mention of certain link")
+    public void verifyGoogleSearch() {
+
+        GoogleSearchPage googleSearchPage = new GoogleSearchPage();
+        googleSearchPage
+                .performSearch(SEARCH_REQUEST)
+                .checkInSearchResult(MENTION);
+
+        assertTrue(driver
```

Comment:
extract into variable, it could be useful during debug

`var currentUrl = driver.getCurrentUrl();`